main.ts

```diff
@@ -38,18 +48,29 @@ const defs = [
   babelDef,
   typescriptDef,
   esProposalsDef,
-];
-
-const main = fork(defs) as any as Main;
-export default main;
+]);
 
 export {
-  ASTNode,
-  Type,
   AnyType,
+  ASTNode,
+  astNodesAreEquivalent,
+  builders,
+  builtInTypes,
+  defineMethod,
+  eachField,
   Field,
+  finalize,
+  getBuilderName,
+  getFieldNames,
+  getFieldValue,
+  getSupertypeNames,
+  namedTypes,
```

Comment:
Exporting `namedTypes` here combines all three versions together, though using a named export requires consumers to import `namedTypes` by name.